Adding missing release notes (#2789)
kiegroup/kie-docs: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.41.0.Final/core-case-traceability.adoc

[id='stunner-data-objects-741']

= Core process engine

> * When retrieving information about user tasks, the correlation id of the associated process and the type of process (i.e. process or case instance) is now also returned.

Reviewer: ```suggestion
* When retrieving information about user tasks, the correlation Id of the associated process and the type of process (for example, process or case instance) is now also returned.
```
